pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -1,8 +1,23 @@
 { stdenv, nixosTests, lib, edk2, util-linux, nasm, acpica-tools, llvmPackages
+, fetchurl, python3, pexpect, xorriso, qemu, dosfstools, mtools
 , csmSupport ? false, seabios
 , fdSize2MB ? csmSupport
-, fdSize4MB ? false
+, fdSize4MB ? secureBoot
```

Comment:
I never enabled `fdSize4MB` during my tests. Turning it on solves the hang.
Would you happen to know where that comes from? and why you turned it on?

That would be great to document that, and maybe add an:
```
assert secureBoot -> fdSize4MB;
```